en/docs/install-and-setup/setup/api-controller/importing-apis-via-dev-first-approach.md

```diff
@@ -397,10 +428,9 @@ WSO2 API Controller (**apictl**) allows you to create and deploy APIs without us
             <td>To add custom sequences, save them in XML format and add them to the corresponding folder. For example, to add a custom in-sequence, save the custom sequence as <code>       SampleSequence.xml</code> and add it to the <code>Sequences/in-sequence/</code>directory.</td>
         </tr>
         <tr class="even">
-        <td><pre><code>Docs
-      |── FileContents</code></pre>
+        <td><pre><code>Docs</code></pre>
         </td>
-        <td>Contains the documents. To add a new document, add the document in the <code>Docs/FileContents/</code>directory.</td>
+        <td>Contains the documents.</td>
```

Comment:
Shall we also describe Client-certificates, Endpoint-certificates and Image in this table?

Reply:
+1, will add those

Reply:
Fixed via https://github.com/wso2/docs-apim/pull/2510/commits/99e306966e259b23d2d3bcef02b1842099970345